charts/prometheus-smartctl-exporter/templates/service.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.serviceMonitor.enabled }}
```

Comment:
Maybe make this service.enabled... There could be folks using the service along with a manually installed service monitor or prometheus config.

The rest looks good though. Thanks! :)

Reply:
Thanks @kfox1111 , I agree.  I've added that and leave Service enabled by default just in case.